tests/console/yast2_clone_system.pm

```diff
@@ -0,0 +1,23 @@
+use base "console_yasttest";
+use testapi;
+
+sub run() {
+    my $self    = shift;
+
+    become_root();
+    type_string "PS1=\"# \"\n";
+
+    script_run("rm -f /root/autoinst.xml; zypper -n in autoyast2 ; yast2 clone_system ; echo CLONED >/dev/$serialdev\n");
```

Comment:
assert_script_run wont work because I am trying to interact with autoyast later, but you are right that it can be improved.